libs/ledger-live-common/src/families/bitcoin/wallet-btc/__tests__/wallet.integration.test.ts

```diff
@@ -137,21 +136,4 @@ describe("testing wallet", () => {
     });
     expect(Buffer.from(tx, "hex")).toHaveLength(20);
   });
-
-  it("should throw during sync if there is an error in explorer", async () => {
-    const client = account.xpub.explorer.underlyingClient;
-    // eslint-disable-next-line no-underscore-dangle
-    const _get = client.get;
-    client.get = async () => {
-      throw new Error("coucou");
-    };
-    let thrown = false;
-    try {
-      await wallet.syncAccount(account);
-    } catch (e) {
-      thrown = true;
-    }
-    client.get = _get;
-    expect(thrown).toEqual(true);
-  });
```

Comment:
this unit test is removed as we use "network" in LLC as other families instead of underlyingClient now